dataland-frontend/src/assets/content.json

```diff
@@ -0,0 +1,193 @@
+                        "In Dataland, ",
+                        "you",
+                        " have the power."
+                    ]
+                },
+                {
+                    "title": "Join a campaign",
+                    "text": [
+                        "Join a campaign to",
+                        " get the missing ",
+                        "data.",
+                        "The Proof of Concepts are a simple way for you to get better data coverage and improved data quality, while testing Dataland."
+                    ],
+                    "cards": [
+                        {
+                            "title": "Pathways to Paris",
+                            "text": "Captures companies’ progress on Paris-compatible transformation paths that are necessary to achieve the goals of the Paris climate protection agreement.",
+                            "date": "14th of August"
+                        },
+                        {
+                            "title": "LkSG",
+                            "text": "The Lieferkettensorgfalts-pflichtengesetz is a German law requiring companies to monitor human rights and environmental risks in their supply chains."
+                        },
+                        {
+                            "title": "EU Taxonomy",
+                            "text": "The EU Taxonomy Regulation supports companies to capture their environmentally sustainable economic activities."
+                        },
+                        {
+                            "title": "SFDR",
+                            "text": "SFDR helps investors to properly assess how sustainability risks are integrated in the investment decision process."
+                        }
+                    ]
+                },
+                {
+                    "title": "Get in touch",
+                    "text": [
+                        "Get in touch"
+                    ],
+                    "cards": [
+                        {
+                            "icon": "Email",
+                            "title": "Your message",
+                            "text": "Get in touch",
+                            "date": "By clicking Get in Touch,  you agree to our Terms of use and that you have read our Privacy Policy."
+                        },
+                        {
+                            "icon": "/static/images/Erik_Breen.jpg",
+                            "title": "Erik Breen,",
+                            "text": "Managing Director of Dataland"
+                        },
+                        {
+                            "icon": "erik.breen@dataland.com",
+                            "title": "I am interested in Dataland’s campaigns 📣 ",
+                            "text": "Dear Erik,\n \nI am sick and tired of data gaps and bad data quality. Therefore, I am very much interested in the SFDR Campaign.\n \nPlease keep me updated.\n \nBest regards, "
```

Comment:
Why does it say "in the SFDR Campaign" in the email text if I click on "I AM INTERESTED"?
The way I interpret the page, the "I AM INTERESTED" button is not associated to one specific framework.
So why is the SFDR framework explicitly contained in the mail text?

Reply:
I also wonder the same... these are the info I got from our UX designer, I'll gonna ask her to check it and give me a feedback

Reply:
Ok thanks. Then let's wait for feedback, because this does not make sense in my opinion.

Reply:
Fixed with the new version